apps/src/code-studio/pd/workshop_enrollment/workshop_enrollment.jsx

```diff
@@ -135,6 +137,11 @@ export default class WorkshopEnrollment extends React.Component {
   }
 
   renderSuccess() {
+    analyticsReporter.sendEvent(EVENTS.WORKSHOP_ENROLLMENT_COMPLETED_EVENT, {
+      'regional partner': this.props.workshop.regional_partner.name,
```

Comment:
Is it possible `workshop.regional_partner` is null? If so, should we use optional chaining via `?.`?

Reply:
Good call!